jobs/multi-arch-pipeline.Jenkinsfile

```diff
@@ -244,6 +244,7 @@ recipe:
   git_ref: ${params.STREAM}
   git_url: https://github.com/${repo}
   git_commit: ${params.FCOS_CONFIG_COMMIT}
+copy-build: ${parent_version}
```

Comment:
Will gangplank correctly handle the empty string case? (Which obviously won't happen very often, just when starting up new streams, but it's good to be resilient to it.)

Reply:
yep - it handles that.